tests/link/jax/test_as_jax_op.py

```diff
@@ -0,0 +1,474 @@
+import jax
+import jax.numpy as jnp
+import numpy as np
+import pytest
+
+import pytensor.tensor as pt
+from pytensor import as_jax_op, config, grad
+from pytensor.graph.fg import FunctionGraph
+from pytensor.link.jax.ops import JAXOp
+from pytensor.scalar import all_types
+from pytensor.tensor import TensorType, tensor
+from tests.link.jax.test_basic import compare_jax_and_py
+
+
+def test_two_inputs_single_output():
+    rng = np.random.default_rng(1)
+    x = tensor("x", shape=(2,))
+    y = tensor("y", shape=(2,))
+    test_values = [
+        rng.normal(size=(inp.type.shape)).astype(config.floatX) for inp in (x, y)
+    ]
+
+    def f(x, y):
+        return jax.nn.sigmoid(x + y)
+
+    # Test with as_jax_op decorator
+    out = as_jax_op(f)(x, y)
+    grad_out = grad(pt.sum(out), [x, y])
+
+    fg = FunctionGraph([x, y], [out, *grad_out])
+    fn, _ = compare_jax_and_py(fg, test_values)
```

Comment:
We merged a change in main, `compare_jax_and_py` no longer expects a FunctionGRaph as input, just inputs, outputs, test_values